hedera-node/src/main/java/com/hedera/services/state/migration/ReleaseTwentySevenMigration.java

```diff
@@ -48,9 +51,9 @@ public static MerkleMap<EntityNum, MerkleStakingInfo> buildStakingInfoMap(Addres
 			final var info = new MerkleStakingInfo(bootstrapProperties);
```

Comment:
```
		if (stakingPeriod != DEFAULT_STAKING_PERIOD_MINS) {
			minStakePerNode = 0L;
		}
```
Also, this is only for our testing

Reply:
Need to be removed before merging the branch to master

Reply:
Good point. 👍